curriculum/challenges/english/01-responsive-web-design/css-nutrition-label/step-040.md

```diff
@@ -0,0 +1,139 @@
+---
+id: 615f6823d0815b7a991f2a75
+title: Step 40
+challengeType: 0
+dashedName: step-40
+---
+
+# --description--
+
+The `float` styling is causing the new `p` element to be outside of the label's border. Add a divider after the `p` element.
```

Comment:
This might just be me, but I interpret this to mean _add an `hr`_.

Maybe extend this to include the words "use your existing..."

Reply:
Pushed a commit up to tweak this.